### ISSUE TITLE
all very cpu intensive.

### ISSUE THREAD

**Issue by ofpcarnage** [Genoil/ZECMiner#74]
0.6 now for me worse than the others.
I have to restart the rig just to get the cards going again. Every 5 mins
I've had to let the pc decide what page file to use to run all 6 gpu's ...currently having to use 9216MB 
6xRX480 8gb
windows 10
Intel Xeon(R) CPU E3-1231 v3 @ 3.40GHz 
8gb Ram
ASRock HTC Pro MB

**Comment by cmos99** [comment]
yeah it is for about 2 minutes then it subsides. 0.6 is slightly lower yield but at least its not crashing

**Comment by ofpcarnage** [comment]
I'd consider not being able to restart a GPU unless a reboot a crash my friend

**Comment by cmos99** [comment]
my setup is identical to yours, except i am running 16GB ram and an I5. Does it just fall over or hang?

Also 16GB pagefile it used to be an eth rig.

**Comment by ofpcarnage** [comment]
So I need to increase my ram maybe ? If so then its cool I have a extra stick of ram available as I was going to build a 2nd rig so I can just use thaT

Edit dang it I have a stick of 1333 and a stick of 1600 mhz  both 8gb

**Comment by ofpcarnage** [comment]
For me its individual cards ...after a while they go to 0.0 and when I restart they wont fire up
Its mainly gpu#0 that goes 1st
